AirCasting/Utils/CLLocationUtils.swift

```diff
@@ -0,0 +1,35 @@
+// Created by Lunar on 14/06/2021.
+//
+
+import CoreLocation
+
+extension CLLocationCoordinate2D: Equatable { }
+
+extension CLLocationCoordinate2D {
+    public init?(latitude: Double?, longitude: Double?) {
+        guard let lat = latitude, let long = longitude else { return nil }
+        self.init(latitude: .init(lat), longitude: .init(long))
+    }
+}
+
+public func ==(lhs: CLLocationCoordinate2D, rhs: CLLocationCoordinate2D) -> Bool {
+    return lhs.latitude == rhs.latitude && lhs.longitude == rhs.longitude
```

Comment:
iMO this is a bad assumption. The accuracy of double is lots and lots and lots wider than the coordinates got from the gps. Comparing this values would probably fail most of the times in some places just because of conversion to double. Assume some rounding.

Reply:
this was a leftover after some experimentation, nice find, removed 👌